tests/tests_geomstats/test_poincare_polydisk.py

```diff
@@ -33,21 +33,17 @@ def test_signature(self, n_disks, expected):
         self.assertAllClose(metric.signature, expected)
 
     @tests.conftest.np_autograd_and_torch_only
-    def test_product_distance_extrinsic_representation(
-        self, n_disks, point_a_extrinsic, point_b_extrinsic
-    ):
+    def test_product_distance(self, point_a_extrinsic, point_b_extrinsic):
```

Comment:
in a previous commit I removed `n_disks` from here. but following @YannCabanes comment, we can now edit this test to receive `n_disks` and then we verify `distance_n_disks = ((n * (n+1) / 2) ** 0.5 * distance_single_disk`.

Reply:
OK - this is done. In theory it is now possible to use `n_disks` polydisks, each factor having `m_disks`, but the smoke data as it is only has points for `m_disks=1` so I've left it there. This now runs with `n_disks=2` and `n_disks=3`.